packages/browser/src/transports/fetch.ts

```diff
@@ -51,6 +51,7 @@ export function makeFetchTransport(
         pendingBodySize -= requestSize;
         pendingCount--;
         return {
+          response,
```

Comment:
TBD if we re-use the transport, but we'll need access to the response body (as well as changing http req method to GET)